profiles/common/modules/custom/tmgmt_poetry/README.md

```diff
@@ -155,20 +119,71 @@ DGCOMM (see 'Installations step 1').
+Once the module is enabled and the settings properly setup, CEM will proceed
+with the module's configuration.
+
+Edit the translator labelled "DGT Connector (auto created)".
+
+In order to do this, navigate to:
+
+
+    Configuration > Regional and Language > Translation management Translator
+
+or go to:
+
+
+    admin/config/regional/tmgmt_translator/manage/poetry_en 
+
+ - Translator settings: [x] Auto accept finished translations
+   - Check this if you don't want to review a translation before publishing it.
+
+ - Translator plugin:
+   - This is shown for information, please do not change it.
+
+ - DGT Connector plugin settings:
+   - You should see 'Main "poetry_service" variable is properly set.' if you
+   have correctly followed the steps above. Otherwise, get back and check what
+   you forgot!
+   - Counter: you do not need to fill this. The counter is auto generated.
+   - Requester code: must always be *WEB* [See DGT reference explanation]
+   (#dgt-reference-number).
+   - Organization responsible, Author & requester: check the example provided.
+   - Contact usernames: should be the *username* (you connect to the network,
+  ecas or the proxy with your username) of the persons in charge of the request.
+  This is important as only these persons can view translation details in the
+  web app.
+   - DGT contacts: The email address of your contact person at DGT. They will
+   receive feedback on the translations you want to comment. That information
+   should have been provided to you by DGCOMM.
+   - Remote languages mappings: Here you should map language codes that are not
+   supported by DGT-Connector to the corresponding language code supported.
+   Typical example is when the site is configured for using 'Portugues from 
+   Portugal' (code pt-pt) that should be mapped to 'pt'.
+
+[Go to top](#table-of-content)
+
+
 # Testing
-:black_joker: There are 2 ways to test the poetry service.
 
-- Locally, without the need to access the webservice  by using the
- tmgmt_poetry_mock module.
+## Testing locally (for developers)
 
- See [the mock readme] (tmgmt_poetry_mock/README.md) for more information.
+If you hired a contractor and he needs to test the DGT-Connector UI and 
+workflow locally, this can be done without the need to access the webservice by 
+using the tmgmt_poetry_mock module.
 
-- On Playground:
+ Contractors : see [the mock readme] (tmgmt_poetry_mock/README.md) for more 
+ information on testing with the mock.
 
- 1) Follow the [requesting instance procedure](#requesting-access-to-poetry).
+## Testing inhouse (for webmasters)
+When the DGT-Connector will be properly enabled, setup and configured in your 
+playground environment, CEM will ask you to perform few tests monitored by DGT.
 
- 2) Await confirmation that the [Playground was configured by the maintenance
- team](#enabling-the-feature-on-your-platform-instance).
+Go to next section ([Usage](#usage)) on how to complete your test.
```

Comment:
Why not "Go to the [Usage](#usage) section"?... Or at least, add a "the" before "next section".